### ISSUE TITLE
Win32 install fails, because no JRE gets installed?

### ISSUE THREAD

**Issue by zspitzer** [utdream/lucee-installer#88]
just trying lucee-5.2.4.037-pl0-windows-installer.exe on an old 32 bit server, 
it's failing because there's no JRE being installed? only the jdk is installed

Looks like it's getting confused checking for the JRE? 

```
Script output:
 The JAVA_HOME environment variable is not defined correctly
This environment variable is needed to run this program
NB: JAVA_HOME should point to a JDK not a JRE
```

```
Creating directory D:\lucee\jre32-win
Creating directory D:\lucee\jre32-win\jdk
Creating directory D:\lucee\jre32-win\jdk\bin
Creating directory D:\lucee\jre32-win\jdk\bin\dtplugin
Creating directory D:\lucee\jre32-win\jdk\bin\plugin2
Creating directory D:\lucee\jre32-win\jdk\bin\client
Creating directory D:\lucee\jre32-win\jdk\lib
Creating directory D:\lucee\jre32-win\jdk\lib\deploy
Creating directory D:\lucee\jre32-win\jdk\lib\management
Creating directory D:\lucee\jre32-win\jdk\lib\cmm
Creating directory D:\lucee\jre32-win\jdk\lib\i386
Creating directory D:\lucee\jre32-win\jdk\lib\applet
Creating directory D:\lucee\jre32-win\jdk\lib\images
Creating directory D:\lucee\jre32-win\jdk\lib\images\cursors
Creating directory D:\lucee\jre32-win\jdk\lib\security
Creating directory D:\lucee\jre32-win\jdk\lib\jfr
Creating directory D:\lucee\jre32-win\jdk\lib\fonts
Creating directory D:\lucee\jre32-win\jdk\lib\ext

Creating directory D:\lucee\jre64-win
Creating directory D:\lucee\jre64-win\jdk
Creating directory D:\lucee\jre64-win\jdk\include
Creating directory D:\lucee\jre64-win\jdk\include\win32
Creating directory D:\lucee\jre64-win\jdk\jre
Creating directory D:\lucee\jre64-win\jdk\jre\bin
Creating directory D:\lucee\jre64-win\jdk\jre\bin\server
Creating directory D:\lucee\jre64-win\jdk\jre\lib
Creating directory D:\lucee\jre64-win\jdk\jre\lib\amd64
Creating directory D:\lucee\jre64-win\jdk\jre\lib\management
Creating directory D:\lucee\jre64-win\jdk\jre\lib\cmm
Creating directory D:\lucee\jre64-win\jdk\jre\lib\applet
Creating directory D:\lucee\jre64-win\jdk\jre\lib\images
Creating directory D:\lucee\jre64-win\jdk\jre\lib\images\cursors
Creating directory D:\lucee\jre64-win\jdk\jre\lib\security
Creating directory D:\lucee\jre64-win\jdk\jre\lib\jfr
Creating directory D:\lucee\jre64-win\jdk\jre\lib\fonts
Creating directory D:\lucee\jre64-win\jdk\jre\lib\ext
Creating directory D:\lucee\jre64-win\jdk\bin
Creating directory D:\lucee\jre64-win\jdk\lib
```

**Comment by utdream** [comment]
Hi Zac,

Yeah, that doesn't look good. It does look like the JRE is getting installed based on your logs, (Creating directory D:\lucee\jre64-win\jdk\jre) but I'm guessing it's not getting implemented properly. Need to check it.

Can you tell me what version of Windows you were installing to? You mentioned 32-bit... Win 8 maybe? Not supporting Win XP any longer because it's not supported by MS. If you ARE using XP I'll go ahead and check it against Win 8 32-bit and see if I can duplicate the issue there.

Thank you for finding this!

**Comment by zspitzer** [comment]
ahaha, it was a client's old, retired 2003 server... i installed the jdk manually and replaced the
lucee installed folder with the jdk folder and things worked....

I'm guessing the 32bit jdk-jre zip file is just missing the jre? I wanted to check but couldn't guess the command line to get the installer just to unpack itself to have a look... is there a command line switch to make it just unpack itself?

**Comment by utdream** [comment]
IIRC you can run a --help option on the installer to get the available options, but I don't think there's anything as specific as unpacking certain files in those options. I just checked the source files that I used to build the installers, and the 32-bit JRE is there, so it was probably just something I did to screw it up. I'll check it on a Windows 8 32-bit machine and get it worked out.

Thanks again for bringing this to my attention. Have a great weekend!

**Comment by utdream** [comment]
Hi Zac,

Okay, so I spent just over an hour today attempting to duplicate this issue, and the current installer worked on every windows OS I tried it on. It's possible I already fixed this issue or maybe it's just an issue with the unsupported OS. I'm not sure!

If you try this on a supported OS (an OS that MS actively supports) and you encounter anything like this again, just reopen this issue (or open another one) and I'll be happy to take a look at this again.

Thanks again for reporting this and merry Christmas to you and yours!
